### DISCUSSION TITLE
How do I do medvram in vlad diffusion? Can someone explain?

### DISCUSSION THREAD

**Discussion by superprohero** [vladmandic/sdnext#847] [Q&A]
![bvjhabdvjhbhja](https://github.com/vladmandic/automatic/assets/133063465/89ba6efd-596b-40d1-a657-c8b6e9522a66)

This is how it used to work in automatic 1111, but how does this work in Vlad diffusion?

**Comment by vladmandic** [comment]
same. create that file and single line `webui --medvram` is enough - you can see that rest of lines are not doing much (add --xformers as well if you really want them).

**Reply by superprohero** [reply]
Wait, so should I just do this?
![vxdvdssvs](https://github.com/vladmandic/automatic/assets/133063465/e53273c7-4a4c-45a3-9df0-9a384863cfd0)

**Reply by brknsoul** [reply]
Woah.. no. Don't edit webui.bat.
Since you're using Windows, use the method I described below.

**Reply by superprohero** [reply]
so here I should type what you said?

![drrgedrgrege](https://github.com/vladmandic/automatic/assets/133063465/32422892-b179-49ac-b9ea-52942de14dcb)

**Reply by brknsoul** [reply]
Leave the first part alone (the stuff that's high-lighted in blue), add a space after the final double quotes ("), and then add `--medvram`. Then click OK, and run SD using this new shortcut.

That field should now look like this;
`"C:\Stable diffusion\automatic\webui.bat" --medvram`

**Reply by superprohero** [reply]
"C:\Stable diffusion\automatic\webui.bat" --medvram

I will copy and paste this without the "quote" symbol, ok?
Thanks for your help, man.

edit

![](https://user-images.githubusercontent.com/1969381/237248319-a7976172-e3f9-4c2c-b823-3a042c82813e.png)

"quote" symbol or not?

I understand now, like this right

![drrgedrgrege](https://github.com/vladmandic/automatic/assets/133063465/8a0c0889-fb76-4f27-b928-7ea780dc1f22)

**Reply by brknsoul** [reply]
No. You need the quotes. Your field should look like; 
`"C:\Stable diffusion\automatic\webui.bat" --medvram` 
Include the quotes.

My image doesn't use quotes because there's no spaces in my Target. There are spaces in yours (eg: `C:\Stable diffusion...`).

**Reply by superprohero** [reply]
Thanks for the help, man. Sorry for my stupidity.^_^

**Reply by superprohero** [reply]
![drrgedrgrege](https://github.com/vladmandic/automatic/assets/133063465/d8ccdffc-d25e-4eea-a375-310f891ecaeb)

 like this right

**Reply by brknsoul** [reply]
Correct.

**Reply by superprohero** [reply]
cool I don't notice any speed difference

**Reply by brknsoul** [reply]
`--medvram` just lowers the likelihood of Out of Memory errors while sacrificing a minor amount of speed, not that noticeable. However, you can use `--lowvram` for very low memory graphics cards (2gb, for example), but this has a significant loss of generation speed.

In Windows, I use `--medvram` because Windows likes to hog the GPU memory a little for desktop management and Explorer.
In Linux, I don't need to do this, because there's not much using GPU memory.

If you're using `--medvram` in Windows, ensure that your browser, mail client and Discord aren't using Hardware Acceleration mode. You can google how to turn these off per application. (eg: google "discord disable hardware acceleration").

**Reply by superprohero** [reply]
Despite the 8 GB of VRAM, 1440x1440 was not possible in Hires fix. That's why I went to medvram. Also, thanks for giving me a tip for Windows. You are really helpful, man.

**Comment by brknsoul** [comment]
In Windows you can right-click webui.bat and click "Create shortcut". Right-click this new shortcut and click "Properties", you can then add your options in the Target field;
![image](https://github.com/vladmandic/automatic/assets/1969381/a7976172-e3f9-4c2c-b823-3a042c82813e)

**Reply by superprohero** [reply]
so here I should type what you said?
![drrgedrgrege](https://github.com/vladmandic/automatic/assets/133063465/3d6d64d2-7d69-45af-88bd-a722dabee441)
